cmd/auth/login.go

```diff
@@ -56,6 +60,38 @@ var loginCmd = &cobra.Command{
 			return err
 		}
 
+		if configureCluster {
+			err := root.MustWorkspaceClient(cmd, args)
+			if err != nil {
+				return err
+			}
+			ctx := cmd.Context()
+			w := root.WorkspaceClient(ctx)
+
+			promptSpinner := cmdio.Spinner(ctx)
+			promptSpinner <- "Loading names for Clusters drop-down."
+			names, err := w.Clusters.ClusterInfoClusterNameToClusterIdMap(ctx, compute.ListClustersRequest{})
+			close(promptSpinner)
+			if err != nil {
+				return fmt.Errorf("failed to load names for Clusters drop-down. Please manually specify required arguments. Original error: %w", err)
+			}
+			clusterId, err := cmdio.Select(ctx, names, "The cluster to be attached")
```

Comment:
This message is printed at upon selecting one. It should also make sense in this context.